geps/gep-91.md

```diff
@@ -0,0 +1,22 @@
+# GEP-91: Client Certificate Validation for TLS terminating at the Gateway Listener
+
+* Issue: [#91](https://github.com/kubernetes-sigs/gateway-api/issues/91)
+* Status: Provisional
+
+(See definitions in [GEP Status][/contributing/gep#status].)
+
+## TLDR
+
+This GEP proposes a way to validate the TLS certificate presented by the downstream client to the server
+(Gateway Listener in this case) during a [TLS Handshake Protocol][], also commonly referred to as mutual TLS (mTLS).
+
+## Goals
+- Define an API field to specify the CA Certificate within the Gateway Listener configuration that can be used as a trusted anchor to validate the certificates presented by the client.
```

Comment:
Isn't the existing `certificateRefs` field where the CA certificate would be specified? Is the idea that this GEP will add a field that can be used to specify something like a client validation cacert key in a `cacertificateRefs` secret, for example? Maybe adding an API section with a proposal would be helpful to start the discussion?

Reply:
fwiw, this goal of adding a way to config mTLS termination in a Gateway is valuable and very contained scope IMO.

Reply:
Agree that feels like the inevitable placement here. Let's get this in and then we can work on the implementation details.